aws/provider.go

```diff
@@ -910,6 +910,7 @@ func init() {
 		"kinesis_analytics",
 		"kinesis",
 		"kinesisanalytics",
+		"kinesisvideo",
```

Comment:
Can you please also add this to the available customizable service names in `website/docs/guides/custom-service-endpoints.html.md`? Thanks so much.

The CONTRIBUTING documentation surrounding this should be available soon: https://github.com/terraform-providers/terraform-provider-aws/pull/8277/files#diff-8cc7b2b0d78dd2501610391c086a8516R288

Reply:
@bflad Thanks. Fixed it.